togetherly/lib/providers/person_provider.dart

```diff
@@ -22,21 +22,30 @@ class PersonProvider with ChangeNotifier {
   List<Child> _children = [];
   List<Child> get children => _children;
 
-  Future<void> addPerson(Person person) async {
+  Person? _currentPerson;
+  Person? get currentPerson => _currentPerson;
+
+  bool _ready = false;
+  bool get ready => _ready;
+
+  Future<int?> addPerson(Person person) async {
```

Comment:
Provider methods should not return futures with any values, only `Future<void>`. Otherwise, this creates more than one place where consumers need to listen to state. I see that the only place you use the value returned here is in `profile.dart`, and you use it to update the user identity. That kind of business logic should not be in the UI layer, so you should move it into one of the Providers as a new method (perhaps `createPersonForCurrentUser` in `PersonProvider`?)

Reply:
Done